src/schema/clusterViz.js

```diff
@@ -0,0 +1,49 @@
+const clusterVizSchema = [];
+
+{
+    const name = 'initClusterRequest';
+    const schema = {
+        doc: {
+            name: 'Project Cluster Start Request',
+            description: 'start clustering for project',
+        },
+        fields: {
+            cluster_model_id: { type: 'number', required: true },
```

Comment:
Ask BP to send it in camelcase, add todo for now

Reply:
@bewakes